src/ImportDefinitionsBundle/Command/ExportCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Import Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2016-2018 w-vision AG (https://www.w-vision.ch)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+namespace ImportDefinitionsBundle\Command;
+
+use ImportDefinitionsBundle\Event\ExportDefinitionEvent;
+use Pimcore\Console\AbstractCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use ImportDefinitionsBundle\Model\ExportDefinitionInterface;
+
+final class ExportCommand extends AbstractCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('import-definitions:export')
+            ->setDescription('Run a Export Definition.')
+            ->setHelp(<<<EOT
+The <info>%command.name%</info> runs a Export Definitions and imports Objects.
+EOT
+            )
+            ->addOption(
+                'definition', 'd',
+                InputOption::VALUE_REQUIRED,
+                'Import Definition ID'
+            )
+            ->addOption(
+                'params', 'p',
+                InputOption::VALUE_REQUIRED,
+                'JSON Encoded Params'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws \Exception
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $eventDispatcher = $this->getContainer()->get('event_dispatcher');
```

Comment:
This is deprecated, you should directly inject your dependencies into the constructor.